src/scripts.js

```diff
@@ -71,8 +71,8 @@ function generateCategories() {
 function displayCat(items) {
   homeView.innerHTML = '';
   items.forEach(item => {
-    homeView.innerHTML += `<article class="category">${item}
-    </article>`;
+    homeView.innerHTML += `<div class="container"><article class="category"><img id="glove" src="assets/icons/gloves.svg"/><img class="placeholder-image" src="assets/spices-556x370.jpg" alt="placeholder image"/><h3>${item}</h3>
```

Comment:
I don't anticipate this change causing any issues, but if we notice anything we'll revisit.